query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -13,11 +13,48 @@ func init() {
 	plan.RegisterPhysicalRules(
 		PushDownRangeRule{},
 		PushDownFilterRule{},
+		PushDownGroupRule{},
 		//PushDownReadTagKeysRule{},
 		PushDownReadTagValuesRule{},
 	)
 }
 
+// PushDownGroupRule pushes down a group operation to storage
+type PushDownGroupRule struct{}
+
+func (rule PushDownGroupRule) Name() string {
+	return "PushDownGroupRule"
+}
+
+func (rule PushDownGroupRule) Pattern() plan.Pattern {
+	return plan.Pat(universe.GroupKind, plan.Pat(ReadRangePhysKind))
+}
+
+func (rule PushDownGroupRule) Rewrite(node plan.Node) (plan.Node, bool, error) {
+	src := node.Predecessors()[0].ProcedureSpec().(*ReadRangePhysSpec)
+	grp := node.ProcedureSpec().(*universe.GroupProcedureSpec)
+
+	// GroupExcept not a supported
+	if grp.GroupMode == flux.GroupModeExcept {
```

Comment:
Instead checking for the unsupported group mode, maybe look for the group modes that _are_ supported, and return otherwise?  That way if another group mode is added, we won't accidentally push it down.